### ISSUE TITLE
Allow multi-line string in `var.json`

### ISSUE THREAD

**Issue by felipecrs** [hassio-addons/bashio#94]
# Problem/Motivation

This was the goal, so that users could set something like:

```yaml
# Add-on configuration.yaml

additional_sip_config: |
  [my-custom]
  config = yes
```

## Expected behavior

```
bashio::var.json \
    auto_add "^$(bashio::config 'auto_add')" \
    persons "^${persons}" \
    additional_config "$(bashio::config 'additional_sip_config')" 
``` 

To produce a valid JSON with properly escaped line breaks and so.

## Actual behavior

```
2022/01/04 18:28:46 invalid character '\n' in string literal
```

## Steps to reproduce

I think the command above should be enough to replicate the issue.

## Proposed changes

None yet.

**Comment by felipecrs** [comment]
A bit off-topic, but it would be nice if the add-on configuration UI could support multiline strings too.

**Comment by felipecrs** [comment]
I'll try to take a look at this.

**Comment by felipecrs** [comment]
Proposed a fix at #104

**Comment by felipecrs** [comment]
Still valid.
